packages/components/dropdown/src/DropdownTrigger.tsx

```diff
@@ -17,7 +17,7 @@ interface TriggerProps {
 const styles = cva(
   [
     'flex w-full cursor-pointer items-center justify-between',
-    'min-h-sz-44 rounded-lg bg-surface px-lg',
+    'min-h-sz-44 rounded-lg bg-surface text-on-surface px-lg',
```

Comment:
We forgot to specify the text color, so whenever the Dropdown was used inside a `success-container` for example, it has a `on-success-container` text color, etc